src/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -54,6 +54,15 @@ To mitigate these problems, SDK version 7.54.0 will [truncate console messages](
 
 If you're having any problems with the latest SDK version, we want to hear about it. Please open a [GitHub issue](https://github.com/getsentry/sentry-javascript/issues/new?assignees=&labels=Type%3A+Bug&projects=&template=bug.yml) and describe your situation.
 
+## Using Session Replay increases the bundle size of my application.
+
+Enabling Session Replay will add about 50 kb (gzipped) to your application bundle.
+While this is a significant increase, we believe the benefits of Session Replay outweigh the cost.
+Due to the inherent complexity of the browser enviornment, there is a considerable amount of code necessary in order for Session Replay to work.
+
+While we are generally working on ways to reduce the bundle size, there are a few steps you can take right now in order to reduce the size of Session Replay,
+based on your specific use case. See [Tree Shaking](../../configuration/tree-shaking/) for more information.
+
 ## I see the message: "A large number of mutations was detected (N). This can slow down the Replay SDK and impact your customers."
```

Comment:
```suggestion
## I See the Message: "A large number of mutations was detected (N). This can slow down the Replay SDK and impact your customers."
```